qemu/tests/balloon_check.py

```diff
@@ -52,10 +52,13 @@ def get_ballooned_memory(self):
         :rtype: int
         """
         try:
-            output = self.vm.monitor.info("balloon")
-            ballooned_mem = int(re.findall(r"\d+", str(output))[0])
-            if self.vm.monitor.protocol == "qmp":
-                ballooned_mem = ballooned_mem / (1024 ** 2)
+            if self.vm.monitor:
+                output = self.vm.monitor.info("balloon")
+                ballooned_mem = int(re.findall(r"\d+", str(output))[0])
+                if self.vm.monitor.protocol == "qmp":
+                    ballooned_mem = ballooned_mem / (1024 ** 2)
+            else:
+                return 0
```

Comment:
How about add a log before return,  so we can distinguish if the return value is the size of memory.

Reply:
updated